tests/unit/modules/dcnm/test_dcnm_net.py

```diff
@@ -0,0 +1,737 @@
+# Copyright (c) 2020-2023 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
```

Comment:
Same here to be consistent can we use the full name for this file `test_dcnm_network.py`?

Reply:
Changes done as suggested.